### PR TITLE
Resolve coupling between objects in `\Magento\QuoteGraphQl\Model\Cart\SetBillingAddressOnCart`

### PR DESCRIPTION
**Pull request by vasilii-b**
### Description (*)
Fixed issue magento/graphql-ce#468

### Fixed Issues (if relevant)
1.  magento/graphql-ce#468

### Manual testing scenarios (*)
1. Go to `/dev/tests/static/` (CLI `cd /dev/tests/static/` from project root folder)
2. Prepare your `phpunit.xml` from `phpunit.xml.dist`
3. run `../../../vendor/bin/phpunit`
4. Tests results should be green

image

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/QuoteGraphQl/Model/Cart/CreateQuoteAddressByCustomerAddress.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        $this->getCustomer = $getCustomer;
+        $this->getCustomerAddress = $getCustomerAddress;
+    }
+
+    /**
+     * @param ContextInterface $context
+     * @param int|string|null $customerAddressId
+     * @param array|null $customerAddress
+     * @return Address
+     * @throws GraphQlAuthenticationException
```
Comment (TomashKhamlai) [inline app/code/Magento/QuoteGraphQl/Model/Cart/CreateQuoteAddressByCustomerAddress.php]: Usually `@throws` annotation tags are added only when exception is thrown explicitly in the function. Any other exceptions from parent methods are implicit and not annotated. "Missing `@throws` tag(s)" can be turned off in PHPStorm Inspections->PHP->PHPDoc

**Comment by TomashKhamlai** [comment]
@naydav. Checked. Works as expected

File: app/code/Magento/QuoteGraphQl/Model/Cart/CreateQuoteAddressByCustomerAddress.php (hunk 1 above)
Comment (vasilii-b) [reply app/code/Magento/QuoteGraphQl/Model/Cart/CreateQuoteAddressByCustomerAddress.php]: Thanks for mentioning, @TomashKhamlai. I've added requested changes. Thank you!

Comment (naydav) [reply app/code/Magento/QuoteGraphQl/Model/Cart/CreateQuoteAddressByCustomerAddress.php]: > Usually @throws annotation tags are added only when exception is thrown explicitly in the function

I little disagree with this statement. If I work with some API I will prefer to know which of exception could be thrown without submerging in implementation (does not matter will be thrown in current method or in some child classes)

Generally, it's about readability.

**Pull request merged**